zmon_aws_agent/main.py

```diff
@@ -163,6 +165,7 @@ def main():
     current_entities = (
         elbs + scaling_groups + apps + application_entities + rds + elasticaches + dynamodbs + certificates)
     current_entities.append(aws_limits)
+    current_entities.append(sqs)
```

Comment:
I would also add ``'sqs': sqs`` to debug output https://github.com/zalando-zmon/zmon-aws-agent/pull/99/files#diff-17074aed459fb7aacbb29f90d801d63aR202